org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/binding/wizard/project/ProjectBindingWizard.java

```diff
@@ -202,9 +206,19 @@ public boolean performFinish() {
         p.deleteAllMarkers(SonarLintCorePlugin.MARKER_REPORT_ID);
         SonarLintCorePlugin.clearIssueTracker(p);
         JobUtils.notifyServerViewAfterBindingChange(p, oldServerId);
-        SonarLintUiPlugin.subscribeToNotifications(p);
+        projectToSubscribeToNotifications.add(p);
       }
     });
+    if (!projectToSubscribeToNotifications.isEmpty()) {
+      Job subscribeToNotificationsJob = new Job("Subscribe to notifications") {
```

Comment:
This job is duplicated at several places, we could have a NotificationsSubscribeJob ?

Reply:
:+1: